content/_code-samples/use-tickets/js/use-tickets-multisig.js

```diff
@@ -0,0 +1,129 @@
+              Account: wallet_3.classicAddress,
+              SignerWeight: 1,
+            },
+          }
+        ],
+        SignerQuorum: 2,
+      }
+
+    const signerLiSetSignerList_tx_prepared = await client.autofill(signerLiSetSignerList_tx)
+    const SetSignerList_tx_signed = main_wallet.sign(signerLiSetSignerList_tx_prepared)
+    console.log("\n SignerListSet Tx hash:", SetSignerList_tx_signed.hash)
+  
+    const setsignerlist_submit = await client.submitAndWait(SetSignerList_tx_signed.tx_blob)
+    console.log("           Submit result:", setsignerlist_submit.result.meta.TransactionResult)
+    
+    const CreateTicket_tx = await client.autofill({
+      "TransactionType": "TicketCreate",
+      "Account": main_wallet.address,
+      "TicketCount": 3
+    })
+  
+    const CreateTicket_tx_signed = main_wallet.sign(CreateTicket_tx)
+    console.log("\n CreateTicket Tx hash:", CreateTicket_tx_signed.hash)
+  
+    const ticket_submit = await client.submitAndWait(CreateTicket_tx_signed.tx_blob)
+    console.log("        Submit result:", ticket_submit.result.meta.TransactionResult)
+    
+    const ticket_response = await client.request({
+        "command": "account_objects",
+        "account": main_wallet.address,
+        "ledger_index": "validated",
+        "type": "ticket"
+    })
+
+    console.log(`\n- Tickets issued by ${main_wallet.address}:\n`)
+    for (var i = 0; i < ticket_response.result.account_objects.length; i++) {
+        console.log(`Ticket ${i+1}: ${ticket_response.result.account_objects[i].TicketSequence}`)
+    }
+
+    // We'll use this ticket on our tx
+    const ticket_1 = ticket_response.result.account_objects[0].TicketSequence
+    
+    console.log(`\n Ticket sequence ${ticket_1} will be used for our multi-sig transaction`)
+    
+    const Payment_tx = {    
+        "TransactionType": "AccountSet",
+        "Account": main_wallet.address,
+        "TicketSequence": ticket_1,
+        "LastLedgerSequence": null,
+        "Sequence": 0
+    }
+
+    const Payment_tx_prepared = await client.autofill(Payment_tx, signersCount=3)
+    
+    // Each signer will sign the prepared tx (AccountSet_tx) and their signatures will be combines into 1 multi-sig transaction
+    const { tx_blob: Payment_tx_signed_1 } = wallet_1.sign(Payment_tx_prepared, multisign=true)
+    const { tx_blob: Payment_tx_signed_2 } = wallet_2.sign(Payment_tx_prepared, multisign=true)
+    const { tx_blob: Payment_tx_signed_3 } = wallet_3.sign(Payment_tx_prepared, multisign=true)
+  
+    console.log("\n All signers have signed the transaction with their corresponding keys")
+
+    // Combine 3 of the signers' signatures to form a multi-sig transaction
+    const multisignedTx = xrpl.multisign([Payment_tx_signed_1, Payment_tx_signed_2, Payment_tx_signed_3])
+
+    const multisig_submit = await client.submitAndWait(multisignedTx)
+    console.log("\n Multi-sig Submit result:", multisig_submit.result.meta.TransactionResult)
```

Comment:
nit: log the hash for the successful transaction like you did for the other two transactions.  That way it can be looked up in explorers.